hoomd/mpcd/CollisionMethod.cc

```diff
@@ -238,6 +266,216 @@ void mpcd::CollisionMethod::storeInitialEmbeddedGroupVelocities(uint64_t timeste
         }
     }
 
+void mpcd::CollisionMethod::thermalizeConstituentParticles(uint64_t timestep)
+    {
+    // zero accumulators
+    m_linmom_accum.zeroFill();
+    m_angmom_accum.zeroFill();
+
+    Scalar T_set(1.0); // for debugging purposes
```

Comment:
Per discussion on Slack, let's move the temperature to be stored in the base class. Then, require it to be not null if there are rigid bodies participating in the collision in the base class.